docs/01-route-registrar-usage.md

```diff
@@ -78,6 +84,11 @@ The route-registrar expects a configuration json file like the one below:
   - `route_service_url` is optional. When provided, Gorouter will proxy
     requests received for the `uris` above to this address.
   - `health_check` is optional and explained in more detail below.
+  - `terminate_frontend_tls` is optional. When true, the router will terminate 
+    TLS before forwarding the requests to the backend servers. Default: false
+  - `enable_backend_tls` is optional. When true, the router will initiate a 
+    TLS connection to the backend servers. Default: false 
+  - `alpns` is optional and is an array of Application Layer Protocol Negotiation strings.
```

Comment:
ALPNs is required if terminate_frontend_tls is set, right? If so can that get reflected here?

Reply:
After going through the rest of the PR it sounds like it ALPNs are optional even if terminate_frontend_tls is enabled